tools/cabana/cabana.cc

```diff
@@ -33,7 +33,9 @@ int main(int argc, char *argv[]) {
     replay_flags |= REPLAY_FLAG_QCAMERA;
   }
 
+#ifndef __APPLE__
   OpenpilotPrefix op_prefix;
+#endif
```

Comment:
we should just make the prefix work with ZMQ. @deanlee want to try to implement that?

Reply:
I'll try it later, let's keep the #ifndef for now.